CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+
+And you don't just have to write code. You can help out by writing documentation, tests, or even by giving feedback about this work. (And yes, that includes giving feedback about the contribution guidelines.)
+
+(Thanks to [Adrienne Lowe](https://github.com/adriennefriend/imposter-syndrome-disclaimer) who came up with this disclaimer and made it OPEN for anyone to use!)
+
+## How can I contribute?
+
+There are multiple ways in which you can contribute:
+
+### Reporting a Bug
+
+Firefly is in active development. It's no surprise that you may encounter something that doesn't work for your use case. Or maybe you have some suggestions about how we can improve some functionality. Feel free to share any of it with us by [opening an issue](https://docs.github.com/en/github/managing-your-work-on-github/creating-an-issue) at [Firefly Github Issues](https://github.com/Caltech-IPAC/firefly/issues).
+
+Please make sure that you provide all the necessary information, especially how to reproduce your problem — it will not only make our work easier but will also help you communicate your problem more effectively.
+
+### Editing the Documentation
+
+Due to the rapid pace of development and limited resources, Firefly's documentation may not always be up-to-date or fully comprehensive. And here lies an opportunity for you: you can edit the documentation stored as markdown text files (`*.md`) in the [`docs` directory](https://github.com/Caltech-IPAC/firefly/tree/dev/docs) of Firefly and as docstrings in the `src` code files.
+
+You can see the Firefly JS docs at http://localhost:8080/firefly/docs/js/index.html if you have built them locally, as follows:
+```shell
+gradle firefly:buildJsDoc
+gradle firefly:buildAndDeploy
+```
+
+After editing the markdown or source files, build the docs again to check if your changes render correctly. Then you can submit your changes to us by making a patch as described in the next section.
+
+### Making a Patch (aka Code Contribution)
+
+Currently, Firefly tracks development priorities and tasks through Jira, which is internal to us, rather than through public GitHub issues. As a result, we don't have "good-first" or "easy" labeled issues to invite code contributions from new contributors. However, that shouldn't discourage you from contributing!  As a new user, you have a better perspective on where our documentation lacks, and you can edit it to make it better, as described in the previous section.
+
+If you've [set up Firefly locally](https://github.com/Caltech-IPAC/firefly/tree/dev?tab=readme-ov-file#setup) and have some familiarity with the codebase, we encourage you to contribute by adding features you'd find useful or fixing issues that affect your workflow. Your contributions will be invaluable to improve open-sourced Firefly for other users as well.
```

Comment:
Firefly differs here compared to other open sourced software, I tried to write it to communicate that limitation to non-IPAC/Rubin contributors while sharing ways in which they can make PRs

Reply:
I think this is fine for now.